apollo-api/src/commonMain/kotlin/com/apollographql/apollo/api/ExecutionContext.kt

```diff
@@ -8,46 +8,110 @@ import kotlin.jvm.JvmField
  * A context of GraphQL operation execution, represented as a set of [Key] keys and corresponding [Element] values.
  */
 @ApolloExperimental
-class ExecutionContext private constructor(private val context: Map<Key<*>, Element>) {
+interface ExecutionContext {
```

Comment:
will be rebased after merge this PR: https://github.com/apollographql/apollo-android/pull/2258